### PR TITLE
[v3.2.1] Adding webhook ports

### PR DESCRIPTION
**Pull request by tomahawk28**
Toward https://github.com/giantswarm/giantswarm/issues/15920#issuecomment-791324333

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Comment by tomahawk28** [comment]
Tested in `geckon`

Status update:
```
 curl --location --request GET 'https://app-operator-3-1-0.g8s.geckon.gridscale.kvm.gigantic.io/'
<html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>nginx</center>
</body>
</html>
```

Metrics:
```
curl --location --request PATCH 'https://app-operator-3-1-0.g8s.geckon.gridscale.kvm.gigantic.io/status/00000/chart-operator-unique' \
                   --header 'Authorization: cvcvcvcvcv' \
                   --header 'Content-Type: application/json' \
                   --data-raw '{
                       "status": "success"
                   }' -i
HTTP/2 200
date: Fri, 05 Mar 2021 14:56:18 GMT
content-type: application/json; charset=utf-8
content-length: 5
strict-transport-security: max-age=15724800; includeSubDomains

null
```

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+## [3.2.1] - 2021-03-05
+
+### Security
+
+- stop exposing `/metrics` endpoints through ingress object.
```
Comment (rossf7) [inline CHANGELOG.md:14]:
```suggestion
- Restrict ingress to only expose the status endpoint.
```

I think better to describe this the other way round. WDYT?

File: helm/app-operator/templates/ingress.yaml [hunk 2]
```diff
@@ -18,7 +18,7 @@ spec:
     - host: {{ include "resource.default.name" . }}.{{ .Values.Installation.V1.Kubernetes.API.Address }}
       http:
         paths:
-          - path: /
+          - path: /status
```
Comment (rossf7) [inline helm/app-operator/templates/ingress.yaml:21]: One line fixes are the best fixes. ❤️

Ping @cokiengchiara so you don't miss out. 😄

**Review by rossf7 — APPROVED** [review]
LGTM Thanks for taking care <3

**Comment by tomahawk28** [comment]
I will change the version and will merge.

File: pkg/project/project.go [hunk 3]
```diff
@@ -5,7 +5,7 @@ var (
 	gitSHA      = "n/a"
 	name        = "app-operator"
 	source      = "https://github.com/giantswarm/app-operator"
-	version     = "3.2.0"
+	version     = "3.2.1"
```
Comment (tomahawk28) [inline pkg/project/project.go:8]: Updating the version as 3.2.1

**Pull request merged**